src/assets/styles/sass/styles.scss

```diff
@@ -0,0 +1,3 @@
+body {
+    background-color: rgb(233, 12, 233);
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/descholar-ceo/portfolio/pullRequest?prid=5731031)